locales/de/common.json

```diff
@@ -57,77 +57,77 @@
-  "Fix": "reparieren",
+  "Associate with NXM URLs": "NXM URLs verwalten",
+  "Fix nxm link on Chrome (Requires Chrome to be closed)": "NXM-Links für Chrome reparieren (Chrome darf nicht laufen)",
+  "Fix": "Reparieren",
   "Apply": "Anwenden",
   "Is Chrome running?": "Läuft Chrome gerade?",
   "Cancel": "Abbrechen",
   "Continue": "Fortsetzen",
   "Chrome has to be closed, otherwise this fix has no effect.": "Chrome muss geschlossen sein, sonst hat dieser Fix keinen Effekt.",
-  "Nothing changed": "Keine Änderung",
+  "Nothing changed": "Keine Änderungen vorgenommen",
   "No change was necessary": "Es waren keine Änderungen nötig",
-  "Failed to fix NXM handling in Chrome": "Konnte Behandlung von NXM links in Chrome nicht reparieren",
+  "Failed to fix NXM handling in Chrome": "Konnte Verwendung von NXM-Links in Chrome nicht reparieren",
   "Report": "Melden",
   "Success": "Erfolg",
   "Fix was applied": "Erfolgreich repariert",
   "Categories": "Kategorien",
   "Plugins": "Plugins",
   "Save Games": "Spielstände",
   "Show downloads from other games": "Downloads für andere Spiele anzeigen",
-  "Savegame id": "Spielstand id",
+  "Savegame id": "Spielstand-ID",
   "Creation Time": "Zeitpunkt d. Speicherns",
   "Delete": "Löschen",
   "Browse": "Suchen",
   "Meta server": "Metaserver",
-  "Servers to query for meta data.": "Server auf denen nach Metadaten gesucht wird.",
+  "Servers to query for meta data.": "Server, auf denen nach Metadaten gesucht wird.",
   "Add a metaserver": "Metaserver hinzufügen",
   "{{count}} selected": "{{count}} ausgewählt",
-  "Do you really want to delete this mod?": "Wollen Sie diese Mod löschen?",
-  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods löschen?",
+  "Do you really want to delete this mod?": "Wollen Sie diese Mod wirklich löschen?",
+  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods wirklich löschen?",
   "Remove Mod": "Mod löschen",
   "Remove Archive": "Archiv löschen",
   "Disable Dependent": "Abhängige Mods deaktivieren",
-  "Confirm deletion": "Löschung bestätigen",
-  "Confirm": "Fortfahren",
+  "Confirm deletion": "Löschen bestätigen",
+  "Confirm": "Bestätigen",
   "Third-party libraries": "Bibliotheken von Drittanbietern",
-  "License": "Lizens",
+  "License": "Lizenz",
   "Released under": "Veröffentlicht unter",
   "External Changes": "Externe Änderungen",
-  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Dateien wurden geändert seit Vortex zuletzt aktiviert hat. Sie müssen entscheiden was damit passiert.",
+  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Eine oder mehrere Dateien wurden seit dem letzten Start von Vortex verändert. Sie müssen bestimmen, was mit ihnen passiert.",
```

Comment:
Nicht korrekt, der Fehler ist seit dem letzten "Deployment" (aka "Aktivierung") aufgetreten, nicht seit dem letzten Start.

Reply:
Dann ersetze ich das durch "seit dem letzten Deployment" oder so. Wäre vielleicht gut, das im Englischen auch noch irgendwann zu ändern, weil ich "activate" jetzt sonst noch nie als Synonym zu "deploy" gesehen habe.

Reply:
Ich habe mal ein bisschen zusammengeschrieben: https://forums.nexusmods.com/index.php?/topic/6539436-translating-vortex/
Da ist eine Anleitung dabei wie man unübersetzte Texte aufspürt, das sollte helfen wenn du die Übersetzung aktualisieren willst.

Reply:
Habe das versucht, geht bei mir aber nicht, weil, auch bei der neuesten Version, die Option nicht im Menü steht. Ging zwar bis jetzt auch ohne, aber ich bräuchte es schon, um rauszufinden, ob manche Sachen vielleicht in andere Dateien gehören.